spec/system/search/display_articles_search_spec.rb

```diff
@@ -1,16 +1,13 @@
 require "rails_helper"
```

Comment:
The setup/mocking of objects for this spec had to change. For search, we use a new `tsvector` column called `reading_list_document`. We use DB triggers to keep this updated. The `Article` factory relies on the front matter in `body_markdown` so to satisfy validations and the new search logic (we also search against `cached_tag_list` now), I switched this all up.